team4_pii.py

```diff
@@ -16,3 +16,10 @@ def find_twitter_usernames(text):
     if match:
         return True
     return False
+
+
+def find_email_handle(text):
+    match = re.search(r'^([^.@]+)(\.[^.@]+)*@([^.@]+\.)+([^.@]+)$', text)
```

Comment:
Keep in mind this won't match something like
```
email address: joe@apple.com
```
Which may or may not be an issue for you in sprint-4 depending on how your split up your input string. No need to fix this, just pointing it out.